lib/css/CssParser.js

```diff
@@ -1024,7 +1029,7 @@ class CssParser extends Parser {
 		}
 
 		module.buildInfo.strict = true;
-		module.buildMeta.exportsType = "namespace";
+		module.buildMeta.exportsType = this.namedExports ? "namespace" : "default";
```

Comment:
What if we put it in generator options, but read and use it here, just thought named exports sematically is more about generator things then parser. It is about more user experience than a real code.

Reply:
Read generator options in parser feels a bit wired.

namedExports decide how the parent module import it, the source code of the parent module should be `import style from './style.css'` or `import * as style from './style.css'`, from the 'source code' aspect parser options make sense to me.

I can change this if you think generator options is more make sense 😃

Reply:
@ahabhgk Yeah, I thought about it and studied the code and I think you're right, the only thing that confuses me is the user experience, but css is still experiment, so if developers will faced with problems we will try to improve our docs